style/Makefile

```diff
@@ -20,5 +21,16 @@ config.js:
 code_format:
 	npx prettier --write .
 
-run: sprites config.js
+build/maplibre-js:
+	mkdir -p build/maplibre-js
+	git clone git@github.com:wipfli/maplibre-gl-js.git build/maplibre-js
+
+build/maplibre-js/dist/maplibre-gl.js: build/maplibre-js
+	cd build/maplibre-js; git checkout 270a47f473ba51375a7df4076ee271777cc04467; \
+          npm ci && npm run build-prod-min;
```

Comment:
I will note that this installs way too much crap that isn't needed to simply build the prod js and css, but this is of course temporary until the named commit is merged and released.

Reply:
That's just local/temporary, right?  I assume this doesn't deploy node_modules.

Reply:
Yes of course, just idly complaining.

(Java? Ah, for some diagramming tool in the docs.)